.gitlab/pipeline/zombienet.yml

```diff
@@ -12,4 +12,4 @@ include:
   # polkadot tests
   - .gitlab/pipeline/zombienet/polkadot.yml
   # bridges tests
-  - .gitlab/pipeline/zombienet/bridges.yml
+  # - .gitlab/pipeline/zombienet/bridges.yml
```

Comment:
this is temporary?

Reply:
Yes, let me leave a comment. Actually, we **may** do a hack here and release relayer that uses that branch earlier than this PR is merged. But I feel that we may lost track of what relayer and where it is used. So IMO better to comment and uncommet soon after (I hope minutes/hours after) this PR and #4157 will be merged

Reply:
https://github.com/paritytech/polkadot-sdk/pull/4102/commits/18e15fb42e92fa4d1b92fe32cda567e8fc5eadf8